src/config.rs

```diff
@@ -15,7 +16,7 @@ pub struct Config {
     pub db_password: String,
 }
 
-impl Config {
+impl AppConfig{
```

Comment:
неправильное оформление кода
```suggestion
impl AppConfig {
```